src/components/CurrencyName/index.js

```diff
@@ -2,7 +2,7 @@ import React from 'react'
 import PropTypes from 'prop-types'
 
 // Current mapping does not contain any logic
-export const outcomeTokenToText = () => 'Outcome Token'
+export const outcomeTokenToText = outcomeToken => outcomeToken
```

Comment:
A function which returns the same as it receives as a parameter? Why do we need a function for that?

Reply:
This was put in place for when we implement loading the outcome Tokens name from the Blockchain. It's still not needed, it could just as well be made a string. But not in the Backmerge